src/main/java/com/mraof/minestuck/client/renderer/entity/ServerCursorRenderer.java

```diff
@@ -34,4 +37,10 @@ public RenderType getRenderType(ServerCursorEntity animatable, ResourceLocation
 	{
 		return RenderType.entityTranslucentCull(getTextureLocation(animatable));
 	}
+	
+	@Override
+	protected void applyRotations(ServerCursorEntity animatable, PoseStack poseStack, float ageInTicks, float rotationYaw, float partialTick)
+	{
+		poseStack.mulPose(Axis.YP.rotationDegrees(180.0F - animatable.getYRot()));
+	}
```

Comment:
Does this have to do with the change from LivingEntity to Entity for ServerCursorEntity?

Reply:
Yeah, it does. `GeoEntityRenderer` only handles render rotations for `LivingEntity` inheritors, so the change required us to do so manually.